.github/workflows/deploy-stg-ecs.yml

```diff
@@ -59,30 +59,28 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           NPM_BASE_64_AUTH: ${{ secrets.NPM_BASE64_AUTH }}
           NPM_EMAIL: ${{ secrets.NPM_EMAIL }}
-      - uses: mr-smithers-excellent/docker-build-push@v5
-        name: build and push to ECR Legacy DOTCOM
-        with:
-          image: docs-worker-pool-legacydotcomstg
-          tags: latest
-          registry: ${{ steps.login-ecr.outputs.registry }}
-          dockerfile: Dockerfile.legacy
-          buildArgs: NPM_EMAIL=${{ secrets.NPM_EMAIL }},NPM_BASE_64_AUTH=${{ secrets.NPM_BASE64_AUTH }}
-        env:
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          NPM_BASE_64_AUTH: ${{ secrets.NPM_BASE64_AUTH }}
-          NPM_EMAIL: ${{ secrets.NPM_EMAIL }}
+      # - uses: mr-smithers-excellent/docker-build-push@v5
```

Comment:
I think we want to keep this section uncommented. It's needed for the subsequent deploy steps

Reply:
Since it uses the legacy dockerfile, in its current state it isn't compatible with my changes. Should I change dockerfile and/or the image name?

Reply:
Oh, my apologies. I misunderstood this section. You can just delete this uncommented section entirely as it's just for the legacy dockerfile like you said.

Reply:
I delete it in DOP-4306 as it falls under the scope of that ticket technically- but it needs to be commented out here not to cause errors which is why I just left it commented out

Reply:
Okay, sounds good to me! It's fine as is then